apps/mobile/src/features/send/send-sheet.tsx

```diff
@@ -1,11 +1,19 @@
 import { SheetNavigationContainer } from '@/common/sheet-navigator/sheet-navigation-container';
 import { useSheetNavigatorContext } from '@/common/sheet-navigator/sheet-navigator-provider';
 import { FullHeightSheet } from '@/components/full-height-sheet/full-height-sheet';
+import { SendSheet as RefactoredSendSheet } from '@/features/send-refactor/send-sheet';
+import { useBoolVariation } from '@launchdarkly/react-native-client-sdk';
 
 import { SendSheetNavigator } from './send-sheet-navigator';
 
 export function SendSheet() {
   const { sendSheetRef } = useSheetNavigatorContext();
+  const sendRefactorEnabled = useBoolVariation('send_refactor', false);
```

Comment:
I think it's now or never. Lets make this change now and remove old code while we're at it or else not make the change